File: OOP C++/lab2/Drop.cpp
#include "Drop.h"
#include <stdexcept>
#include "FactoryInitializer.h"

void Drop::Execute(Operands& operands, Tokens& tokens, std::string& output, Reader& reader) {
    operands.GetAndPop();
}

namespace {
    FactoryInitializer<Drop> Registration("drop");
}

File: Lab1/src/Commands/Drop.java
package Commands;

import Utility.Operands;

import java.util.StringTokenizer;

public class Drop implements Command{
    @Override
    public void Execute(StringTokenizer tokens, Operands operands) {
        operands.GetAndPop();
    }

    @Override
    public void Check(StringTokenizer tokens, Operands operands) {

    }

}